### PR TITLE
new inspect that outputs ruby code

### PR DESCRIPTION
**Pull request by dancinglightning**
ok second try. to_sexp as was and new inspect.
sorry for misunderstanding before

### CODE REVIEW

File: lib/ast/node.rb [hunk 1]
```diff
@@ -211,7 +211,33 @@ def to_sexp(indent=0)
 
       sexp
     end
-    alias :inspect :to_sexp
```
Comment (whitequark) [inline lib/ast/node.rb]: This should become `alias :to_s :to_sexp`.

**Comment by whitequark** [comment]
By the way, you don't have to recreate pull requests. Any commits you add or replace in your branch will be picked up by GitHub.

File: lib/ast/node.rb (hunk 1 above)
Comment (dancinglightning) [inline lib/ast/node.rb]: to_s is defined, so i did't want to touch that
figured you can easily do that line if you want it. That change hasn't really got anything to do with the inspect changing.

**Comment by dancinglightning** [comment]
btw, thanks, i did know. But i had changed the tests beyond recognition, so it was easier to start fresh

**Comment by whitequark** [comment]
Merged in 84da2668452.

**Pull request closed without merging**